### DISCUSSION TITLE
希望新增功能，视频双语字幕快捷方式给个开关，可以选择在哪些网站开启

### DISCUSSION THREAD

**Discussion by itianmi** [immersive-translate/immersive-translate#2221] [Ideas]
### 新功能 | New Feature

希望新增功能，视频双语字幕快捷方式给个开关，可以选择在哪些网站开启。

### 为什么您想要添加这些功能 | Why you want to add these features

我在油管等国外视频网站上，有时候需要视频双语字幕的，但在国内网站是不需要这功能的。
如果打开“YouTube 自动开启双语字幕”，我不需要的时候关闭就比较麻烦。
如果打开“在播放器显示双语字幕快捷方式”，国内的网站又多出一个完全用不到的按钮。

### 提议 | Proposal

希望新增功能，视频双语字幕快捷方式给个开关，可以选择在哪些网站开启。

**Comment by theowenyoung** [comment]
可以这样，把快捷方式关掉，然后可以在 popup 面板上操作是否开启双语字幕
